.bash_prompt

```diff
@@ -164,7 +164,15 @@ sexy_bash_prompt_branch_exists () {
 sexy_bash_prompt_parse_git_ahead () {
   # Grab the local and remote branch
   branch="$(sexy_bash_prompt_get_git_branch)"
-  remote_branch="origin/$branch"
+  
+  #remote_branch="origin/$branch"
+  #Compute the remote tracking branch, if no remote branch then echo 0
+  branchnamefull=$(git symbolic-ref -q HEAD);
+  remote_branch=$(git for-each-ref --format='%(upstream:short)' $branchnamefull)
```

Comment:
I should also mention that we want to avoid `for-each-ref` as it does pattern matching so if there are any pattern charactes in a branch name then it won't work =/